spec/system/listing_orders_spec.rb

```diff
@@ -1,54 +1,71 @@
 require 'rails_helper'
 
 RSpec.describe 'Listing orders management', type: :system do
-  let(:employee) { create(:employee) }
   let(:admin_user) { create(:admin_user) }
-  let!(:kudo) { create(:kudo, receiver: employee) }
-  let(:reward) { create(:reward) }
-  let!(:order) { create(:order, employee: employee, reward: reward) }
+  let!(:order) { create(:order) }
 
   before do
-    login_employee(employee)
+    login_employee(order.employee)
     visit orders_path
   end
 
-  it 'shows employee\'s orders list' do
-    within("li[test_id='order_#{order.id}") do
-      expect(page).to have_text(order.reward_snapshot.title)
-      expect(page).to have_text(order.reward_snapshot.price)
-      expect(page).to have_text(order.reward_snapshot.description)
-      expect(page).to have_text(order.created_at.strftime('%d-%m-%Y'))
-    end
-  end
+  context 'when listing orders' do
+    let(:order_delivered) { create(:order, employee: order.employee, status: 1) }
```

Comment:
`employee: order.employee ` looks a bit weird in the arguments, I would simply create an employee, and pass it as and argument for both orders - just like you did previously - it's not an issue, leave it as it is if you like it :)

```suggestion
    let(:order_delivered) { create(:order, employee: order.employee, status: :delivered) }
```